### ISSUE TITLE
"parser-culture" processing instruction not working / ignored?

### ISSUE THREAD

**Issue by creativesuspects** [richard-scryber/scryber.core#85]
**Describe the bug**
I've added the `parser-culture` processing instruction to my HTML template, but dates are still formatted with the wrong culture (English).

**To Reproduce**
Using Scryber.Core 5.0.7
Add following processing instruction to the HTML template:

```
<?scryber parser-culture="nl-nl" ?>
```

The template contains the following HTML:

```
<time date-format="d MMMM yyyy" datetime="{@:Model.InvoiceDate}" />
```

Also tried it with "nl-NL" and some random other language "fr-fr", same results: English culture is still being used.

![image](https://user-images.githubusercontent.com/161342/204374748-9a33e526-e37c-4fa6-9652-916c5eaac964.png)

**Expected behavior**
The parser-culture should be used to properly format the dates and numbers.

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: Edge
 - Version: 107.0.1418.56

**Comment by richard-scryber** [comment]
Hey Arjan, I will definitely take a look at this. 

To confirm the bound date is a string in the local Dutch format and you would like the output as per the format. 

As a workaround to temporarily solve the issue have you tried passing the invoice date as a datetime vale?

R

**Comment by creativesuspects** [comment]
Hi Richard, Model.InvoiceDate is actually already a DateTime value. I have not tried passing it in as a string value. When I just output the value (not in a <time /> element) it looks like this: 11/29/2022 11:03:12 AM.

Numbers (decimals) are also not formatted according to the parser-culture.

**Comment by creativesuspects** [comment]
I think the `data-format` attribute is related to 6.x, in 5.x only `date-format` works. It does apply the correct format, just not the correct culture.

**Comment by richard-scryber** [comment]
Ah, the parser-culture only applies to the template (e.g. 12,500 is 12 and a half or comma decimals). It should be able to set the output culture using the thread culture, I will check too on Thursday when I am back in. 

If it works for you, let me know and I can Add to the documentation. Thanks Arjan

**Comment by creativesuspects** [comment]
So the parser-culture is only used by the parser to help determine how date strings and number strings in the model should be interpreted? So, when the parser-culture is set to nl-nl decimal string values in the model should use a comma (12,50), for en-us these values should use a dot (12.50)?

I think I already tried to set the thread culture, but I believe that also caused some issues when parsing the PDF. I will try again tomorrow and I’ll let you know how it went.

**Comment by richard-scryber** [comment]
Hi Arjan, I did some testing on Thursday, and can confirm, the v6 beta is working with culture on the current thread (as I remember there was a limitation in .net Core 5 on culture setting.

I have added a unit test into the Expression bindings that creates numbers, dates, times and currencies, which seems to work. So should be ok. Let me know if you still have issues with parsing.

**Comment by creativesuspects** [comment]
Hi Richard, good to know the culture does work in v6 beta. I haven't been able to test that version on my project due to the ImageSharp issue I mentioned earlier (#86).

**Comment by richard-scryber** [comment]
Hi Arjan - I am hoping this is now solved with some recent updates for v6.0 - beta. If you are still involved, then let me know. Otherwise I can hopefully close in a couple of days.
